hathor/manager.py

```diff
@@ -318,6 +318,10 @@ def stop(self) -> Deferred:
             wait_stratum = self.stratum_factory.stop()
             if wait_stratum:
                 waits.append(wait_stratum)
+
+        if isinstance(self.tx_storage, TransactionCacheStorage):
```

Comment:
Why don't you add the `flush()` method to all storages? All others would do nothing while the `TransactionCacheStorage` would.

Reply:
Makes sense, will do it.